components/Header.jsx

```diff
@@ -50,96 +121,231 @@ const useStyles = makeStyles(theme => ({
       textDecoration: 'underline',
     },
   },
+  imageStyle: {
+    verticalAlign: 'middle',
+  },
 }));
 
 const activeStyle = {
-  borderBottom: `1px solid ${colors.primaryFocus}`,
+  borderBottom: `2px solid ${colors.primaryFocus}`,
 };
 
 // TODO: links/routing, mobile
-function Header() {
+const Header = () => {
   const classes = useStyles();
-  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [isDashboardOpen, setIsDashboardOpen] = React.useState(false);
```

Comment:
Let's get rid of the dashboard navbar state since we aren't planning on utilizing it until post-launch